docs/theming/high-contrast-mode.md

```diff
@@ -0,0 +1,245 @@
+---
```

Comment:
This file uses a mix of links with the file extension included and omitted. I don't care which we use, but we should be consistent. 

e.g.: `./colors` vs `colors.md`

Reply:
https://github.com/ionic-team/ionic-docs/pull/3431/commits/5b70e28c09360d0a68a3a02a680f6458ceb4528e